src/account.cpp

```diff
@@ -388,8 +390,7 @@ std::size_t olm::Account::get_unpublished_prekey_json(
         return std::size_t(-1);
     }
     if (current_prekey.published) {
-        last_error = OlmErrorCode::OLM_NO_UNPUBLISHED_PREKEY;
```

Comment:
If we are removing this, it should be removed from the list in `include/olm/error.h` as well